conda/cli/main_search.py

```diff
@@ -240,33 +240,33 @@ def execute_search(args, parser):
             if not context.json:
                 print('%-25s %s  %-15s %15s  %-15s %s' % (
                     disp_name, inst,
-                    pkg.version,
-                    pkg.build,
-                    pkg.schannel,
+                    index_record.version,
+                    index_record.build,
+                    index_record.schannel,
```

Comment:
If we were being properly consistent, as I'd like to be soon, we would use `dist.channel` instead of `index_record.schannel`. But for now this works